File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/TripResponseStartCoordinates.scala

package org.openapitools.client.model


case class TripResponseStartCoordinates (
    _latitude: Option[Double],
    _longitude: Option[Double]
)
object TripResponseStartCoordinates {
    def toStringBody(var_latitude: Object, var_longitude: Object) =
        s"""
        | {
        | "latitude":$var_latitude,"longitude":$var_longitude
        | }
        """.stripMargin
}

File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/TripResponseStartCoordinates.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Start (latitude, longitude) in decimal degrees.
 * @param latitude 
 * @param longitude 
 */
data class TripResponseStartCoordinates (

        @JsonProperty("latitude") val latitude: Double? = null,

        @JsonProperty("longitude") val longitude: Double? = null
) {

}

